runtime/moonbase/src/lib.rs

```diff
@@ -1241,7 +1236,7 @@ impl_runtime_apis! {
 
 	impl nimbus_primitives::AuthorFilterAPI<Block, nimbus_primitives::NimbusId> for Runtime {
 		fn can_author(author: nimbus_primitives::NimbusId, slot: u32) -> bool {
-			<Runtime as pallet_author_inherent::Config>::FullCanAuthor::can_author(&author, &slot)
+			AuthorInherent::can_author(&author, &slot)
```

Comment:
This seems pretty boilerplate, could it be implemented in e.g. a blanket impl?

Reply:
Good question. You mean automatically implement this rutime API for any runtime with the author inherent pallet? Maybe it could, but maybe the `impl_runtime_apis!` macro that is wrapping it would interfere. Have you seen something like that done for runtime apis?